utils_cv/action_recognition/model.py

```diff
@@ -276,14 +328,16 @@ def train_an_epoch(
             else:
                 model.eval()
 
+            # set loader
             dl = data_loaders[phase]
 
+            # collect metrics
             batch_time = AverageMeter()
             losses = AverageMeter()
             top1 = AverageMeter()
             top5 = AverageMeter()
 
-            end = time.time()
+            end = time()
```

Comment:
[minor] Could put code below under the "for step, (inputs,..." line. Otherwise the 02 notebook will not show any sign of life in a long time, plus this does not show in the 01 notebook.
                if step % 10 == 0:
                    print(f"   Phase {phase}: batch {step} of {len(dl)}")

Reply:
done